datajunction-server/datajunction_server/utils.py

```diff
@@ -115,17 +121,20 @@ async def close(self):
 
 
 @lru_cache(maxsize=None)
-def get_session_manager() -> DatabaseSessionManager:
+def get_session_manager(request: Optional[Request] = None) -> DatabaseSessionManager:
     """
     Get session manager
     """
     session_manager = DatabaseSessionManager()
+    session_manager.schema = request.headers.get("tenant")
+    settings = get_settings()
+    settings.customSchema = request.headers.get("new_tenant")
```

Comment:
@sadath-12 from our discussions I remember you telling me that you had to do this so that the alembic script would run properly when you init a new tenant's schema, but since [this](https://github.com/DataJunction/dj/commit/504f198a555b5ef020b35d01ef1ae5c19adf2cad#diff-515aae121be36eeb90e22b78bc3051a911772a4565f2e825def4f17c5da4a4ac) commit the alembic scripts are now configurable to target a specific schema. This means in any of your custom endpoints you could do something like this:

```py
from alembic.config import Config
from alembic import command

...
SCHEMA = request.headers.get("tenant") # For example, "tenant1"
if SCHEMA:
    alembic_cfg = Config()
    alembic_cfg.set_main_option("script_location", "./alembic")
    alembic_cfg.cmd_opts = {"x": [f"uri=postgresql+psycopg://username:password@server:5432/{SCHEMA}"]}
    command.upgrade(alembic_cfg, "head")
...
```

Which means you don't need to add this customSchema to the settings class at all.